docs/developer/release/qa.md

```diff
@@ -122,28 +107,28 @@ REPOSITORY                   TAG         IMAGE ID        CREATED       SIZE
 dangerzone.rocks/dangerzone  <other tag> <different ID>  <newer date>  <different size>
 ```
 
-### 4. Dangerzone successfully installs the container image
+### 2. Dangerzone successfully installs the container image
 
 _(Only for Linux)_
 
 Remove the Dangerzone container image from Docker/Podman. Then run Dangerzone.
```

Comment:
I think we should change this to `dangerzone-machine reset`, so that we reset the Podman state as well.